packages/laravel/src/Support/VueViewFinder.php

```diff
@@ -176,7 +176,9 @@ protected function findVueFiles(string $directory, string $baseDirectory, string
             }
         }
 
-        return $files;
+        return collect($files)
```

Comment:
Shouldn't this only replace slashes in `path`? Also, why not replace `\DIRECTORY_SEPARATOR` directly instead of post-processing the `$files` array?

Reply:
I think the question would be does `scandir` work with `/` on Windows.  Cause if not then you do have to post process the files array so the directory scanning works properly.

Reply:
Yes, `scandir` works with `/` on windows. This snippet worked for me on windows:
```php
$files = [];

foreach (scandir($directory) as $file) {
    if (\in_array($file, ['.', '..'], true)) {
        continue;
    }

    $path = $directory . '/' . $file;

    if (is_dir($path)) {
        $files = array_merge($files, $this->findVueFiles($path, $baseDirectory, $namespace));
    } else {
        if (str($path)->endsWith($this->extensions)) {
            $files[] = [
                'namespace' => $namespace,
                'path' => str_replace(base_path(\DIRECTORY_SEPARATOR), '', $path),
                'identifier' => $this->getIdentifier($path, $baseDirectory, $namespace),
            ];
        }
    }
}

return $files;
```